blocks/text.ts

```diff
@@ -282,7 +279,7 @@ blocks['text_getSubstring'] = {
    * @return {!Element} XML storage element.
    * @this {Block}
    */
-  mutationToDom: function() {
+  mutationToDom: function(this: Block): Element {
     const container = xmlUtils.createElement('mutation');
     const isAt1 = this.getInput('AT1').type === ConnectionType.INPUT_VALUE;
```

Comment:
@cpcallen `this.getInput('AT')` is possible null, because it's added and removed dynamically. Is there a way to guarantee that it's present, or should I just use `!` to assert that it is non-null?